tajo-catalog/tajo-catalog-client/src/main/proto/CatalogProtocol.proto

```diff
@@ -24,53 +24,128 @@ option java_generate_equals_and_hash = true;
 import "CatalogProtos.proto";
 import "PrimitiveProtos.proto";
 
+message GetTablespacesResponse {
```

Comment:
I think that this name is easily confused with GetTablespaceResponse. How about renaming it to ```GetTablespaceListResponse```?